pkg/syncer/fullsync.go

```diff
@@ -274,11 +274,6 @@ func fullSyncDeleteVolumes(ctx context.Context, volumeIDDeleteArray []cnstypes.C
 					continue
 				}
 				if migrationFeatureStateForFullSync {
-					volumeMigrationService, err := migration.GetVolumeMigrationService(ctx, &metadataSyncer.volumeManager, metadataSyncer.configInfo.Cfg)
```

Comment:
volumeMigrationService is initialized in the beginning of this method. Please see full diff